logagg/command.py

```diff
@@ -49,6 +52,12 @@ def define_subcommands(self, subcommands):
         forward_cmd.add_argument('--mongodb-database', help='database to store logs')
         forward_cmd.add_argument('--mongodb-collection', help='collection to store logs')
 
+        forward_cmd.add_argument('--influxdb-server-url', help='DNS of the server whrer influxdb is running')
```

Comment:
@AnushaKonduru There is a typo in the help statement.
I remember telling you many times to use Grammarly.
Not sure why this is happening again and again.

Reply:
@venugopalkandru by mistake I didn't check the code through Grammarly. From next time onwards it won't repeat again.